CMSgov/easi-app
# [EASI-2730] - TRB task list status/buttons/links

# EASI-2730

[Figma](https://www.figma.com/file/iGatgTnlzTuqW9CQFYHDam/TRB-Intake-e2e?node-id=18%3A7105&t=UTxwrrDMUtgI7c2h-0)

## Changes and Description

- Implemented all TRB statuses needed for TRB task list
- Added buttons and links - dependent on varying statuses
- Updated snapshots

Currently, I believe, after initial request from is resubmitted after edits requested, `TRBFeedbackStatus` should move back to `In review`.  BE is checking on this.

Links are placeholders for now until things like Documents and Attendee List are implemented

## How to test this change

Verify all these steps are reflected in the status changes per Figma

- Create TRB request
- Verify Task List
- As admin, request edits - /trb/:trbID/initial-request-form/request-edits
- Verify Task List
- Resubmit initial request form
- Verify Task List (Step 2 should be In Review, BE is working on this)
- For the rest of the steps, will need to manually toggle/adjust `TRBAdviceLetterStatus` , `TRBAttendConsultStatus`, and `TRBConsultPrepStatus,` values to trigger rendering of statuses...

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
